packages/shared/src/components/molecules/activity-info/TokenActivityInformation.svelte

```diff
@@ -1,44 +1,62 @@
 <script lang="ts">
-    import { KeyValueBox } from '@ui'
+    import { type IItems, Table } from '@bloomwalletio/ui'
+    import { FoundryActivity } from '@core/activity'
     import { localize } from '@core/i18n'
     import { IIrc30Metadata } from '@core/token'
-    import { IKeyValueBoxList } from '@core/utils'
     import { getPersistedToken } from '@core/token/stores'
-    import { FoundryActivity } from '@core/activity'
 
     export let activity: FoundryActivity
 
     let metadata: IIrc30Metadata
     $: metadata = <IIrc30Metadata>getPersistedToken(activity.tokenId)?.metadata
 
-    let detailsList: IKeyValueBoxList
-    $: detailsList = {
-        ...(metadata?.name && {
-            tokenName: { data: metadata.name },
-        }),
-        ...(metadata?.symbol && {
-            unit: { data: metadata.symbol },
-        }),
-        decimals: { data: String(metadata.decimals) },
-        ...(metadata?.description && {
-            description: { data: metadata.description },
-        }),
-        ...(metadata?.url && {
-            url: { data: metadata.url, isCopyable: true },
-        }),
-        ...(metadata?.logo && {
-            logo: { data: metadata.logo, isCopyable: true },
-        }),
-        ...(metadata?.logoUrl && {
-            logoUrl: { data: metadata.logoUrl, isCopyable: true },
-        }),
+    const items: IItems[] = []
+    function setItems(activity: FoundryActivity, metadata: IIrc30Metadata) {
```

Comment:
```suggestion
    function setItems(_activity: FoundryActivity, _metadata: IIrc30Metadata) {
```

to not confuse the value inside the function with the component wide one